docs/README.md

```diff
@@ -1,5 +1,46 @@
 # Kernl static documentation site
 
+## Targeting the Material for MkDocs Insiders version
+
+To avoid any breaking change, we target a specific version.
+
+For the preview (`doc/Dockerfile.mkdocstrings`)
+```dockerfile
+FROM ghcr.io/els-rd/mkdocs-material-insiders:version
+
+# Ex.
+FROM ghcr.io/els-rd/mkdocs-material-insiders:4.28.0
+```
+
+In the Github Action workflow (`.github/workflows/deploy-static-site.yml`)
+```yaml
+run: pip install git+https://${{secrets.GH_TOKEN}}@github.com/squidfunk/mkdocs-material-insiders.git@version "mkdocstrings[python]"
+
+# Ex.:
+run: pip install git+https://${{secrets.GH_TOKEN}}@github.com/squidfunk/mkdocs-material-insiders.git@9.0.5-insiders-4.28.0 "mkdocstrings[python]"
+```
+
+### Version upgrade
+
+- Check the [changelog](https://squidfunk.github.io/mkdocs-material/insiders/changelog/) to verify the current version and take into account the breaking changes if any. 
+- Synchronize the fork manually (@pommedeterresautee has the rights).
+- Get the version number from [the changelog](https://github.com/ELS-RD/mkdocs-material-insiders/blob/master/CHANGELOG) of the repo.
+  - Ex.: `mkdocs-material-9.0.5+insiders-4.28.0` (it must respect this pattern).
+- Create a release manually with :
```

Comment:
I don't understand this part. what is creating a release manually ?

Reply:
Whether the fork is manually or automatically synchronized, you have to do the release yourself. The M4M maintainer suggests that we contribute by working on a specific workflow.

While waiting to plan, or not, this work must be done via the interface. Maybe this is what should be specified?

<img width="907" alt="Capture d’écran 2023-01-19 à 09 48 44" src="https://user-images.githubusercontent.com/1962160/213397723-9d50bf3a-67b8-475c-8bc4-802320413271.png">